mmdet3d/core/bbox/coders/centerpoint_bbox_coders.py

```diff
@@ -62,7 +63,7 @@ def _topk(self, scores, K=40):
 
         Args:
             scores (torch.Tensor): scores with the shape of [B, N, W, H].
-            K (int): Number to be kept.
+            K (int): Number to be kept. Defaults to 40.
```

Comment:
@yinchimaoliang change another default like 80 or so? This one should match the general pre_max_size in each task group. Otherwise, probably just no need for a default value.